From: Thomas Moulard <thomas.moulard@gmail.com>
Date: Thu, 13 Jun 2013 11:37:13 +0900
Subject: Look for and compile against Boost.System in CMakeLists.txt.

Look for and compile against Boost.System in CMakeLists.txt.

Author: Thomas Moulard <thomas.moulard@gmail.com>
Bug: https://gforge.inria.fr/tracker/index.php?func=detail&aid=15896&group_id=397&atid=1867
Forwarded: yes
---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a6810c2..e252aa8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -725,7 +725,7 @@ IF(USE_OGRE)
     endif ()
     set(Boost_ADDITIONAL_VERSIONS "1.53.0" "1.52.0" "1.51.0" "1.50.0" "1.49.0" "1.48.0" "1.47.0" "1.46.0" "1.45.0" "1.44.0" "1.44" "1.44.0" "1.42" "1.42.0" "1.41.0" "1.41" "1.40.0" "1.40" "1.39.0" "1.39" "1.38.0" "1.38" "1.37.0" "1.37" )
     # Components that need linking (NB does not include header-only components like bind)
-    set(OGRE_BOOST_COMPONENTS thread date_time)
+    set(OGRE_BOOST_COMPONENTS thread date_time system)
     find_package(Boost COMPONENTS ${OGRE_BOOST_COMPONENTS} QUIET)
     mark_as_advanced(Boost_LIB_DIAGNOSTIC_DEFINITIONS Boost_DIR)
     if (NOT Boost_FOUND)
@@ -733,7 +733,6 @@
       set(Boost_USE_STATIC_LIBS NOT ${Boost_USE_STATIC_LIBS})
       find_package(Boost COMPONENTS ${OGRE_BOOST_COMPONENTS} QUIET)
     endif()
-    find_package(Boost QUIET)
  
     if (Boost_FOUND)
       # Set up referencing of Boost
